Makefile

```diff
@@ -0,0 +1,10 @@
+.PHONY: all build ci-bot clean
```

Comment:
Do we need to create a docker image for ci-bot and then push to a docker registry, e.g. dockerhub or huawei swr?

Reply:
It is ok for me:)  which docker registry is suitable for us?
what about docker.io/Huawei-PaaS/ci-bot ?

Reply:
docker.io/Huawei-PaaS/ci-bot sounds good to me. Besides, we should consider Huawei Cloud SWR as well. Eventually, we will use Huawei Cloud SWR :)

I think we can add docker images support in follow-ups.

Reply:
Understood.